src/marqo/s2_inference/s2_inference.py

```diff
@@ -209,14 +238,14 @@ def _load_model(model_name: str, device: str = get_default_device()) -> Any:
     """
 
     model_properties = get_model_properties(model_name)
-    
+
```

Comment:
add some error handling here in case the model does not load properly

Reply:
create one here https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/s2_inference/errors.py

Reply:
class ModelLoadError(S2InferenceError):
    pass